vwifi.c

```diff
@@ -1,30 +1,30 @@
 #include <linux/module.h>
 
+#include <linux/random.h>
 #include <linux/skbuff.h>
 #include <net/cfg80211.h>
 
 #include <linux/mutex.h>
 #include <linux/workqueue.h>
 
+#include <linux/moduleparam.h>
+
 MODULE_LICENSE("Dual MIT/GPL");
 MODULE_AUTHOR("National Cheng Kung University, Taiwan");
 MODULE_DESCRIPTION("virtual cfg80211 driver");
 
 #define WIPHY_NAME "owl" /* Our WireLess */
 #define NDEV_NAME WIPHY_NAME "%d"
-
-#define SSID_DUMMY "MyHomeWiFi"
-#define SSID_DUMMY_SIZE (sizeof(SSID_DUMMY) - 1)
-
-static u8 fake_bssid[ETH_ALEN] __ro_after_init = {};
+#define SSID_MAX_LENGTH \
+    32 /* According to 802.11 Standard, SSID has max len 32.*/
```

Comment:
Move the comment before the macro definition.

Reply:
fixed in https://github.com/sysprog21/vwifi/pull/4/commits/a02d3d9611e41eb07f7b7a401bffd387324ad764